Phaethon/Core/Model/Login.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Runtime.Serialization;
+
+namespace Core.Model
+{
+    [Serializable]
+    [DataContract]
+    public class Login
+    {
```

Comment:
https://github.com/RaidenRabit/Phaethon/blob/da6d6b8edb567c6e375539f3d112132bbce69993/Phaethon/Core/Model/Login.cs#L13-L23
If Username is unique, why do we need int id? Cant we just use username as id?

Reply:
he could but thats alot of changes to hes current code and its way easyer to have fk to int not name (in total i kinda agree but i beleive its a bit too late to bring this issue up right now)